Makefile

```diff
@@ -84,7 +84,11 @@ else
 endif
 
 ifdef BUILD_QB64
-	EXE ?= qb64
+	ifeq ($(OS),win)
+		EXE ?= qb64.exe
+	else
+		EXE ?= qb64
+	endif
 else
```

Comment:
Looking at the logic this `else` isn't actually a bug, but it's quite odd looking due to the indentation. I'm pretty sure we can change this to `endif` as I intended (and remove the `endif` on line 100) and the logic works just fine.